app/code/Magento/Customer/Controller/Account/ForgotPasswordPost.php

```diff
@@ -40,16 +46,19 @@ class ForgotPasswordPost extends \Magento\Customer\Controller\AbstractAccount im
      * @param Context $context
      * @param Session $customerSession
      * @param AccountManagementInterface $customerAccountManagement
+     * @param LoggerInterface $logger
      * @param Escaper $escaper
      */
     public function __construct(
         Context $context,
         Session $customerSession,
         AccountManagementInterface $customerAccountManagement,
+        LoggerInterface $logger,
```

Comment:
Could you make it last argument and make it optional with fallback to object manager? (needed for backward compatiblity)